### ISSUE TITLE
Blockly Telegram SendTo: Allow for Group chat receivers (not only single users)

### ISSUE THREAD

**Issue by ohuf** [ioBroker/AdapterRequests#99]
_This is the English request following up to this German forum post: [https://forum.iobroker.net/viewtopic.php?f=22&t=3890&p=166023#p166023](url) ._

O.K., so in the Blockly "SendTo" block for a Telegram messsage, there is a "receiver" (Empfänger) field where you can enter the Telegram user id for a specific receiver. The Javascript that's being generated, uses the "user" parameter to the "sendTo" object. The generated code looks like this: 
```
    sendTo("telegram.0", "send", {
        text: 'Woof!',
        user: '123456789'
    });
```
The "user" parameter does not support sending to group chat id's, but the "chatId" parameter does!.
On the other hand, the "chatId" parameter also supports sending to single users!!

**So by exchanging the "user" parameter for the - better - "chatId" parameter in Blockly, you could make admins and users much happier without losing existing functionality.**

The better code would then look like this:

```
    sendTo("telegram.0", "send", {
        text: 'Meeow!',
        chatId: '-987654321'
    });
```

Thanks for considering this change.

Oliver.

**Comment by BuZZy1337** [comment]
Wrong place for that.
I saw you already opened an issue & pull request in the telegram repo. :+1:

**Comment by ohuf** [comment]
Yeah, you're right: I went to the forum, came here.. and then finally found the gist for the Telegram adapter.
After reviewing that file  I decided to fix it by myself. Now waiting for that change to be approved...

**Comment by BuZZy1337** [comment]
As I wrote on your pull request I will check your changes tomorrow. I think we have to do one or two other small changes to keep the whole functionality for "send to multiple users (separated by comma)" in the recipient input field.

**Comment by robson80** [comment]
@ohuf How did you fix this by yourself? It is annoying me all the time I want to create a Telegram related script with Blockly. However, scripting in JS works pretty well (as described in you examples in the first post).

**Comment by BuZZy1337** [comment]
It's implemented in Telegram adapter 1.2.6

See https://github.com/ioBroker/ioBroker.telegram/pull/66

Just enter -1234567 as recipient in the designated inputfield to send the message to the groupchat with the ID - 1234567
